src/components/common/machinesConnectionSelector.jsx

```diff
@@ -30,7 +30,7 @@ export const MachinesConnectionSelector = ({ onValueChanged, loggedUser, connect
         return null;
 
     return (
-        <FormGroup label={_("Connection")} isInline hasNoPaddingTop id={id} className="machines-connection-selector">
+        <FormGroup label={_("Location")} isInline hasNoPaddingTop id={id} className="machines-connection-selector">
```

Comment:
I supported the change because I can see that "connection" can be just as confusing. It's true that connection is the term used by libvirt, but that's it to be honest. It's not really used elsewhere, like in qemu, kvm, or other virtualization technologies.

For a user who didn't really use libvirt before, the term "connection" might be even more confusing. Since if I just google "VM connection" or "Virtualization connection", all the articles are about either connecting to the VM (e.g. ssh, VNC and its alternatives) or about VM's connectivity network. So for a user who didn't use libvirt before, maybe they would confuse "connection" with connectivity to the VM or network connection.

Also what we are referring in the UI as "connection" is not really the whole thing. Libvirt refers to the connection as [driver][+optionalTransportMode]://[pathToLocation]/[system/user], so the examples of "connection" according to libvirt are:
```
qemu:///system
xen:///session
test+tcp://localhost:5000/default
qemu+ssh://root@hail.cloud.example.com/system
```

In our UI, we are using the term "connection" for only 1 part of that, i.e. system/session, and libvirt actually refers to that as "daemon mode", as it specifies under which "mode" (user and its privileges) it will connect to the libvirtd daemon.

Reply:
In short, my point is that I'm open to dropping the term "connection", but also open to trying to figure out alternative term.

Reply:
@garrett this Martin's comment may interest you

Reply:
Also let's probably move this into a separate PR so it doesn't block this, what do you all say?

Reply:
Yes, fully agreed -- that's what I meant by splitting the PRs; this label rename, the auto-naming, and the tabification are all independent from each other. And at least the auto-naming is rather complex.

Maybe "connection" is not ideal, but replacing it with an even less specific name (and breaking translations alongside) isn't that useful either IMHO.